### ISSUE TITLE
Relative Notation

### ISSUE THREAD

**Issue by SambaGodschynski** [werckme/werckmeister#167]
```
[
{
  /baseChord: C11/ -- C13 contains all notes of CMaj
  I II III IV V VI VII I' -- cmaj 
  /baseChord: C11/ -- C-13 contains all notes of CMin
  I II III IV V VI VII I' -- cmin
}
]
```

**Comment by SambaGodschynski** [comment]
could be possible also for chord tracks:
```
/key: Cmin/
Imaj7 | IImin | ...

```

**Comment by SambaGodschynski** [comment]
closed with #360
